spec/swagger_v2/api_swagger_v2_body_definitions_spec.rb

```diff
@@ -6,8 +6,14 @@ module TheBodyApi
       class Endpoint < Grape::API
         resource :endpoint do
           desc 'The endpoint' do
-            params body_param: { type: String, desc: 'param', required: false, documentation: { in: 'body' } }
+            headers XAuthToken: {
+              description: 'Valdates your identity',
+              required: true
+            }
+            params body_param: { type: 'String', desc: 'param', documentation: { in: 'body' } },
+                   'body_string_param' => { type: String, desc: 'string_param', documentation: { in: 'body' } }
```

Comment:
Yeah, and that is not actually not relevant, the problem was that the type was defined as a constant.